docs/src/main/sphinx/admin/properties-query-management.rst

```diff
@@ -32,6 +32,20 @@ stages of a query. You can use the following execution policies:
 The number of partitions to use for processing distributed operations, such as
 joins, aggregations, partitioned window functions and others.
 
+``query.low-memory-killer.policy``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-string`
+* **Default value:** ``total-reservation-on-blocked-nodes``
+
+Configures the behavior to handle killing running queries in the event of low
+memory availability. Supports the following values:
+
+* ``none`` does not kill any queries in the event of low memory.
+* ``total-reservation`` kills the query currently using the most total memory.
+* ``total-reservation-on-blocked-nodes`` kills the query currently using the
+  most memory specifically on nodes that are now out of memory.
```

Comment:
* some value - Verb blah blah.

Need the hyphen and to capitalize the initial letter